Makefile.am

```diff
@@ -372,6 +372,14 @@ perf_inproc_lat_SOURCES = perf/inproc_lat.cpp
 
 perf_inproc_thr_LDADD = src/libzmq.la
 perf_inproc_thr_SOURCES = perf/inproc_thr.cpp
+
+if ENABLE_STATIC
+noinst_PROGRAMS += \
+	perf/benchmark_radix_tree.cpp
```

Comment:
you need to remove .cpp and that should fix the build

Reply:
I tried this, but the build failed with: `no rule to make target 'src/.libs/libzmq.a'`. The cmake build seems to be working though.